packages/components/package.json

```diff
@@ -14,7 +14,6 @@
     "url": "https://github.com/hashicorp/design-system.git",
     "directory": "packages/components"
   },
-  "types": "dist/index.d.ts",
   "license": "MPL-2.0",
   "author": "HashiCorp Design Systems <design-systems@hashicorp.com>",
   "scripts": {
```

Comment:
need to update the script to match the blueprint in order to build types and probably remove npm-run-all

Reply:
The build scripts in this PR do match the blueprints afaict? We could change the lint script to use concurrently instead of npm-run-all in a separate PR if needed as it's a bit separate from the intent of this PR